test_accounts/appealable_issues_test_accounts.md

```diff
@@ -0,0 +1,27 @@
+# Sandbox Test Accounts for Appealable Issues API (v0)
```

Comment:
Similar to my comments on the Legacy Appeals PR, I think the "v0" mention can be excluded from this document, since it should apply to all versions of the API (even though there's only one version for now).